package.json

```diff
@@ -21,7 +21,7 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "react": "^0.13.0"
+    "react": "^0.13.0 || ^0.14.0"
```

Comment:
I'd just increase dep version to `^0.14.0` and go ahead with that, given the fast pace of adoption of new versions in React community.